src/libshadertrap/src/executor.cc

```diff
@@ -1201,41 +1201,142 @@ bool Executor::CheckEqualBuffers(CommandAssertEqual* assert_equal) {
     return false;
   }
 
+  bool result = true;
+  size_t start_index = 0;
+  std::vector<CommandAssertEqual::FormatEntry>& format_entries = assert_equal->GetFormatEntries();
+  const Token t = assert_equal->GetStartToken();
+  std::unique_ptr<Token> tmp(std::move(new Token(t)));
```

Comment:
In modern C++ you should basically never need to use `new`. If you find yourself resorting to `new`. Use `MakeUnique` to make a unique pointer. We have a `MakeUnique` function in this project because we are limited to C++11 and std::make_unique didn't arrive until later versions of C++.

However, I have a different suggestion of how to handle this in any case. I think it would be better to make the placeholder format entry in the parser.